tests/bsim/bluetooth/host/att/timeout/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr HINTS $ENV{ZEPHYR_BASE})
+project(app)
+
+add_subdirectory(${ZEPHYR_BASE}/tests/bluetooth/common/testlib testlib)
+
+target_sources(app PRIVATE
+  ../long_read/bs_main.c
+  ../long_read/bs_sync.c
+  main.c
+)
+
+zephyr_include_directories(
+  ${BSIM_COMPONENTS_PATH}/libPhyComv1/src/
+  ${BSIM_COMPONENTS_PATH}/libUtilv1/src/
+  ../long_read/
```

Comment:
Somewhat related to https://github.com/zephyrproject-rtos/zephyr/pull/79501/files#r1790165096: If we are using things from `long_read`, do we need a separate build for this?

Would it make sense to move whatever we need from `long_read` to the `testlib`?

Reply:
Looking further, it appears some of it is related to the backchannel syncs? Have you looked at the babblekit to see if we can just use that for this? 

If you need something more than what it supports, maybe it can be added to the babblekit?